<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterDialog</class>
 <widget class="RegisterDialog" name="RegisterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>图书管理系统 - 注册</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>用户注册</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <bold>true</bold>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="registerGroupBox">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="usernameLabel">
        <property name="text">
         <string>用户名:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="usernameEdit">
        <property name="placeholderText">
         <string>请输入用户名</string>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="passwordLabel">
        <property name="text">
         <string>密码:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="passwordEdit">
        <property name="placeholderText">
         <string>请输入密码</string>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="confirmPasswordLabel">
        <property name="text">
         <string>确认密码:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="confirmPasswordEdit">
        <property name="placeholderText">
         <string>请再次输入密码</string>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="registerButton">
       <property name="text">
        <string>注册</string>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>取消</string>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>usernameEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>RegisterDialog</receiver>
   <slot>on_usernameEdit_textChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>passwordEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>RegisterDialog</receiver>
   <slot>on_passwordEdit_textChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>confirmPasswordEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>RegisterDialog</receiver>
   <slot>on_confirmPasswordEdit_textChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>registerButton</sender>
   <signal>clicked()</signal>
   <receiver>RegisterDialog</receiver>
   <slot>on_registerButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>RegisterDialog</receiver>
   <slot>on_cancelButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>